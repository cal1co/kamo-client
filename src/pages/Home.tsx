import React from 'react';

function Home() {
    return (
        <div className="home" data-testid="landing-page">
            Hello, World!
        </div>
    )
}

export default Home