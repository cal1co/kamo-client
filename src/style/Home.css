
.landing-page {
    background-color:#f2f2f7;
    height:calc(100vh - 88px);
}