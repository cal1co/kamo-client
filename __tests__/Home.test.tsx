import React from 'react';
import { render, screen, cleanup } from '@testing-library/react';
import Home from '../src/pages/Home';


afterEach(() => {
    cleanup(); 
});

test('that landing page component renders', () => {
    render(<Home/>);
    const homeScreen = screen.getByTestId('landing-page');
    expect(homeScreen).toBeInTheDocument();
    // expect(homeScreen).toMatchSnapshot()
});