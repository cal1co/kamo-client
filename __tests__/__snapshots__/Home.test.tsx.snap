// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`that landing page component renders 1`] = `
<div
  class="home"
  data-testid="landing-page"
>
  Hello, World!
</div>
`;
